[lex.string] Clarify size of string-literal (#5095)
cplusplus/draft: source/lex.tex

are determined by its encoding prefix and sequence of
\grammarterm{s-char}s or \grammarterm{r-char}s
as defined by \tref{lex.string.literal}
where $n$ is the number of encoded code units

> in the result of an evaluation of the \grammarterm{string-literal}
> were it to be evaluated (see below).

Reviewer: ```suggestion
that would result from an evaluation of the \grammarterm{string-literal}
(see below).
```

Author: Applied.
